examples/vcloud-configure-edge/vpn-tunnels.yaml

```diff
@@ -0,0 +1,36 @@
+# Example configuration file for defining VPN tunnels on the vShield Edge Gateway
+#
+# Note that applying this configuration file will replace the current VPN tunnels
+# on the vShield Edge Gateway.
+#
+# Here be dragons:
```

Comment:
There was an explanation for this in the commit message. I've added some more comments in the example vpn config in order to make this more obvious.